addon/ng2/blueprints/ng2/files/src/test/__name___test.ts

```diff
@@ -0,0 +1,21 @@
+import {describe, it, expect, beforeEachProviders, inject} from 'angular2/testing';
+import {<%= jsComponentName %>App} from '../app/<%= htmlComponentName %>';
+
+beforeEachProviders(() => [<%= jsComponentName %>App]);
+
+describe('App: KarmaIntegration', () => {
```

Comment:
if you want to keep this in the blueprints then it should not refer to "KarmaIntegration", you are testing the app.